src/math_eigen.cpp

```diff
@@ -0,0 +1,78 @@
+/* ----------------------------------------------------------------------
```

Comment:
We need to decide if we should put the SPARTA copyright on these files with a note that they are copied from LAMMPS, or if we should keep the LAMMPS copyright, which seems a little weird in SPARTA.